metrics-exporter-prometheus/src/distribution.rs

```diff
@@ -34,9 +34,16 @@ impl Distribution {
     }
 
     /// Creates a summary distribution.
-    pub fn new_summary(quantiles: Arc<Vec<Quantile>>) -> Distribution {
-        let summary = RollingSummary::default();
-        Distribution::Summary(summary, quantiles, 0.0)
+    pub fn new_summary(
+        quantiles: Arc<Vec<Quantile>>,
+        bucket_duration: Duration,
+        bucket_count: u32,
+    ) -> Distribution {
+        Distribution::Summary(
+            RollingSummary::new(NonZeroU32::new(bucket_count).unwrap(), bucket_duration),
```

Comment:
We should actually take a `NonZeroU32` for `bucket_count` rather than just unwrapping here.

Reply:
All fixed up!